Alarmify/Modules/Edit/SubViews/FeaturesListVIew.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FeaturesListVIew.swift
+//  Alarmify
+//
+//  Created by Swastik Patil on 04/06/24.
+//
+
+import SwiftUI
+import AudioToolbox
+
+struct FeaturesListVIew: View {
+    @EnvironmentObject var alarmdata: AlarmData
+    var body: some View {
+        NavigationStack {
+            List {
+                DaySelectionView()
```

Comment:
Rename: DayPickerView. Always follow the same naming style. Selection != Selector != Picker etc.